controllers/toolchaincluster/toolchaincluster_controller.go

```diff
@@ -72,23 +74,77 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	clientSet, err := kubeclientset.NewForConfig(cachedCluster.RestConfig)
 	if err != nil {
 		reqLogger.Error(err, "cannot create ClientSet for the ToolchainCluster")
+		if err := r.updateStatus(ctx, toolchainCluster, clusterNotReadyCondition()); err != nil {
```

Comment:
As Francisc already pointed out in his comment in the previous review, this should be offline as well:
```suggestion
		if err := r.updateStatus(ctx, toolchainCluster, clusterOfflineCondition(err.Error())); err != nil {
```

Reply:
i did understood that previous comment differently , but yea it has to be offline